pyEGA3 has strict version requirements, but included tests run even when
using newer versions.
Author: Orient Kružík (INUITS)
diff -ruN pyega3-4.0.0_orig/pyega3.egg-info/requires.txt pyega3-4.0.0/pyega3.egg-info/requires.txt
--- pyega3-4.0.0_orig/pyega3.egg-info/requires.txt	2021-12-09 14:09:53.000000000 +0100
+++ pyega3-4.0.0/pyega3.egg-info/requires.txt	2022-03-18 16:17:16.080009959 +0100
@@ -1,5 +1,5 @@
-requests==2.26.0
-tqdm==4.19.6
-htsget==0.2.5
-psutil==5.6.6
-urllib3==1.26.7
+requests>=2.26.0
+tqdm>=4.19.6
+htsget>=0.2.5
+psutil>=5.6.6
+urllib3>=1.26.7
diff -ruN pyega3-4.0.0_orig/setup.py pyega3-4.0.0/setup.py
--- pyega3-4.0.0_orig/setup.py	2021-12-08 23:16:24.000000000 +0100
+++ pyega3-4.0.0/setup.py	2022-03-18 16:24:03.075866555 +0100
@@ -33,8 +33,8 @@
     author="EGA team",
     author_email="ega-helpdesk@ebi.ac.uk",
     python_requires=">=3.6",
-    install_requires=["requests==2.26.0", "tqdm==4.19.6", "htsget==0.2.5", "psutil==5.6.6", "urllib3==1.26.7"],
-    tests_require=["pytest~=6.2.5", "coverage==4.5.1", "responses~=0.16.0", "pyfakefs~=4.5.3"],
+    install_requires=["requests>=2.26.0", "tqdm>=4.19.6", "htsget>=0.2.5", "psutil>=5.6.6", "urllib3>=1.26.7"],
+    tests_require=["pytest~=6.2.5", "coverage>=4.5.1", "responses~=0.16.0", "pyfakefs~=4.5.3"],
     keywords=["EGA", "archive"],
     license="Apache License, Version 2.0",
     url="https://github.com/EGA-archive/ega-download-client",
